include/execution.hpp

```diff
@@ -377,33 +514,33 @@ namespace std::execution {
       copy_constructible<remove_cvref_t<_Scheduler>> &&
       equality_comparable<remove_cvref_t<_Scheduler>> &&
       requires(_Scheduler&& __sched, const get_completion_scheduler_t<set_value_t> __tag) {
-        { schedule((_Scheduler&&) __sched) } -> sender_of;
+        { schedule((_Scheduler&&) __sched) } -> sender;
```

Comment:
I can imagine schedulers whose senders are only typed when used in certain contexts. E.g., whether they are cancelable depends on whether the environment's stop token is `unstoppable_token` or not.

Reply:
Oh, wait. `sender_of` doesn't care about `sends_done`. I'll revert this. Nice catch.

Reply:
No wait. That's not right. `sender_of` requires `typed_sender`, and if *any* part of the sender traits potentially depends on the environment, it can't be a typed sender. So the `sender_of` check would fail.

Somewhat unsatisfying.